versioned_docs/version-22.10/releases/centreon-os.md

```diff
@@ -0,0 +1,141 @@
+- [UI] Improved Centreon light and dark themes
+- [UI] Relabeled and improved tooltips for some fields in the Hosts, Host Templates, Host Groups, Services and Service
+  Templates configuration forms
+- [UI] Reworked the banner to be more responsive
+- [UX] Changing themes is now possible by clicking on the profile icon
+- [UX] The quick export button is now available by default for all users that have the admin privilege or the required
+  ACL action access. Deploying a configuration is now simpler for all users.
+- [Resources Status] Manage hosts and services categories (Detail tiles, Listing filter)
+- [Resources Status] Manage hosts and services severities (Logos, level, Detail tiles, Listing filter)
+- [Event logs] Rework CSV exporter in event logs to manage high volumetry
+- [Resources Status] Introduce CSV export in Timeline and Graphs
+- [Resources Status] New Parent Alias column and filter in listing for services
+- [Resources Status] Timeline UI in Resources Status cosmetic enhancements
+- [Anomaly Detection] New type of resources in Resource Status (display and filter)
+- [Anomaly Detection] Prediction enveloppe size management by user from Resources Status
+
+#### Performances
+
+- Move to PHP 8.1
+- The "conf changed" flag in the Poller configuration menu now relies on a faster database query, improving the time to display the page
+
+#### Breaking changes
+
+> The configuration of authorizations via the OpenID Connect protocol has evolved. Automatic addition to a contact
+> group and role management have been improved. It is necessary to review your OpenID Connect configuration.
+
+> In /monitoring/resources API, ‘severity_level’ api return key has been replaced by a json object ‘severity’ containing id, level, name and icon
+
+
+
+## Centreon Collect
+
+### 22.10.0
+
+#### Centreon Engine
+
+##### Enhancements
+
+###### Anomaly Detection improvements
+
+- Recheck has been implemented for Anomaly Detection services, retrieving the actual value of the metric to reevaluate the AD service's status
+- Downtimes on regular services are now propagated to related Anomaly Detection service(s)
+
+###### Settings improvements
+
+- Change the log level of some notification log messages
+- Observations on platforms in the field showed us that the default maximum number of queries per transaction (10000) was too high and caused too much delay, so we set it to 2000 by default
+
+###### Code improvements
+
+- Improved the management of `timed_events` objects
+- Improved the memory management of downtimes
+- Calls to the `atoi()` C function have been replaced by more secure functions
+
+#### Centreon Broker
+
+##### Enhancements
+
+###### Improved communication
+
+- Two new broker streams are available, meant to provide simpler configuration and support of the gRPC protocol.
+- It is now possible to limit the maximum total size of Broker retention files using the `event_queues_total_size` global parameter
```

Comment:
Added the ability to limit the maximum total size of Broker retention files using the `event_queues_total_size` global parameter